Tests/Parser/Client/fixtures/mobile_app.yml

```diff
@@ -2393,3 +2393,15 @@
     type: mobile app
     name: DeepL
     version: ""
+-
+  user_agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.6668.101 Safari/537.36 Language/zh ColorScheme/Light wxwork/4.1.39 (MicroMessenger/6.2) WindowsWechat MailPlugin_Electron WeMail embeddisk wwmver/3.26.39.639 noMediaCs/false
+  client:
+    type: mobile app
+    name: WeWork
```

Comment:
The websites seem to use name "WeCom", not "WeWork". I guess "WeWork" is an old name? We probably should use current one.